<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>JACK Audio Connection Kit - Qt GUI Interface.

   Copyright (C) 2003-2018, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qjackctlSessionForm</class>
 <widget class="QWidget" name="qjackctlSessionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session - JACK Audio Connection Kit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qjackctl.qrc">:/images/session1.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="LoadSessionPushButton">
       <property name="toolTip">
        <string>Load session</string>
       </property>
       <property name="text">
        <string>&amp;Load...</string>
       </property>
       <property name="icon">
        <iconset resource="qjackctl.qrc">:/images/open1.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RecentSessionPushButton">
       <property name="toolTip">
        <string>Recent session</string>
       </property>
       <property name="text">
        <string>&amp;Recent</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SaveSessionPushButton">
       <property name="toolTip">
        <string>Save session</string>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="icon">
        <iconset resource="qjackctl.qrc">:/images/save1.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="SaveSessionVersionCheckBox">
       <property name="toolTip">
        <string>Save session versioning</string>
       </property>
       <property name="text">
        <string>&amp;Versioning</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="UpdateSessionPushButton">
       <property name="toolTip">
        <string>Update session</string>
       </property>
       <property name="text">
        <string>Re&amp;fresh</string>
       </property>
       <property name="icon">
        <iconset resource="qjackctl.qrc">:/images/refresh1.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="InfraClientSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="SessionTreeView">
      <property name="toolTip">
       <string>Session clients / connections</string>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Client / Ports</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>UUID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Command</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="InfraClientWidget">
      <layout class="QGridLayout">
       <item row="0" column="0" rowspan="4">
        <widget class="QTreeWidget" name="InfraClientListView">
         <property name="toolTip">
          <string>Infra-clients / commands</string>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Infra-client</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Infra-command</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="AddInfraClientPushButton">
         <property name="toolTip">
          <string>Add infra-client</string>
         </property>
         <property name="text">
          <string>&amp;Add</string>
         </property>
         <property name="icon">
          <iconset resource="qjackctl.qrc">:/images/add1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="EditInfraClientPushButton">
         <property name="toolTip">
          <string>Edit infra-client</string>
         </property>
         <property name="text">
          <string>&amp;Edit</string>
         </property>
         <property name="icon">
          <iconset resource="qjackctl.qrc">:/images/edit1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="RemoveInfraClientPushButton">
         <property name="toolTip">
          <string>Remove infra-client</string>
         </property>
         <property name="text">
          <string>Re&amp;move</string>
         </property>
         <property name="icon">
          <iconset resource="qjackctl.qrc">:/images/remove1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <tabstops>
  <tabstop>LoadSessionPushButton</tabstop>
  <tabstop>RecentSessionPushButton</tabstop>
  <tabstop>SaveSessionPushButton</tabstop>
  <tabstop>SaveSessionVersionCheckBox</tabstop>
  <tabstop>UpdateSessionPushButton</tabstop>
  <tabstop>SessionTreeView</tabstop>
  <tabstop>InfraClientListView</tabstop>
  <tabstop>AddInfraClientPushButton</tabstop>
  <tabstop>EditInfraClientPushButton</tabstop>
  <tabstop>RemoveInfraClientPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qjackctl.qrc"/>
 </resources>
 <connections/>
</ui>
